presto-main/src/main/java/com/facebook/presto/testing/TestingSession.java

```diff
@@ -54,7 +59,8 @@ private TestingSession() {}
 
     public static SessionBuilder testSessionBuilder()
     {
-        return testSessionBuilder(new SessionPropertyManager());
+        return testSessionBuilder(new SessionPropertyManager(new SystemSessionProperties(),
```

Comment:
Please remove this new constructor that adds the mapping.  Instead, in `SessionPropertyManager`, create a factory method that creates a new `SessionPropertyManager` and just loads up the Java session properties after construction.